pom.xml

```diff
@@ -578,7 +578,7 @@
             <dependency>
                 <groupId>com.microsoft.sqlserver</groupId>
                 <artifactId>mssql-jdbc</artifactId>
-                <version>11.2.1.jre17</version>
+                <version>12.2.0.jre11</version>
```

Comment:
why the jre11 version?

Reply:
There is no JRE17 :)

Reply:
https://github.com/microsoft/mssql-jdbc/issues/2161

Reply:
> Starting with release 12.1.0, we're releasing only two versions. A Java 8 version, and a version for Java 11 and greater. To use driver version 12.2.0 with jre17, please use the 12.2.0jre11 release.